src/transactions/cdp.js

```diff
@@ -391,13 +393,23 @@ export async function openCDP(openingALGOs, openingGARD, commit, toWallet) {
   );
   
   addCDPToFireStore(accountID, -openingMicroALGOs, microOpeningGard, 0);
-  addUserToGleam("openCDP", info.address)
-  
+  await updateTotal(info.address, "totalMinted", microOpeningGard)
+  let user_totals = await loadUserTotals()
+  console.log('totals', user_totals)
+  if(user_totals["totalMinted"] >= 10000000){
+    addUserToGleam("mintGARD", info.address)
+  }
   if (commit) {
     await new Promise(r => setTimeout(r, 1000)); // TODO: More elegant fix (do it in the firestore library)
     updateCommitmentFirestore(info.address, accountID, openingMicroALGOs);
     response.text =
       response.text + "\nFull Balance committed to Governance Period #5!";
+    await updateTotal(info.address, "totalCommitted", openingMicroALGOs)
```

Comment:
My thought process was we await because we want to load the updated user totals on 408 and use them to check if the user should be added to the gleam list.